packages/components/alert/src/components/alert-icon/types.ts

```diff
@@ -1,3 +1,5 @@
 import { HTMLJuloProps } from '@julo-ui/system';
 
-export type AlertIconProps = HTMLJuloProps<'svg'>;
+export interface AlertIconProps extends HTMLJuloProps<'svg'> {
+  placement?: 'left' | 'right';
```

Comment:
If this props is not a required props to be passed, you need to determine the `defaultValue`. Please also make sure if I want to disable the placement then I just need to pass `null`

Reply:
Please add `js-doc` for this props